src/components/ItemCard.vue

```diff
@@ -5,7 +5,7 @@
         <q-avatar size="2rem">
           <q-img :src="item.author.photoURL" />
         </q-avatar>
-        <p class="q-mb-none q-ml-sm text-body1">
+        <p class="q-mb-none q-ml-sm text-body1" v-if="item.author.displayName">
```

Comment:
I already fixed the bug we were having here.

You can `git pull` it from `develop` branch